README_ES.md

```
@@ -0,0 +1,137 @@
+### Validación de la base de datos
+
+Hay un script de validación para cada expansión que comprueba la base de datos en busca de errores comunes. De esa manera intentamos mantener los datos lo más correctos posible, y no olvidarnos de ajustar el campo `NPC.questStarts` cuando ajustamos `Quest.startedBy`. Puedes ejecutar los scripts con:
+
+`lua cli/validate-<expansion>.lua`
+
+Reemplaza `<expansion>` con la expansión que quieras validar (revisa la carpeta `cli` para ver los scripts disponibles).
+
+## Donaciones
+Si quieres apoyar el desarrollo de Questie mediante una donación, puedes hacerlo vía PayPal:
+
+<a href="https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=JCUBJWKT395ME&source=url"><img src="https://www.paypalobjects.com/en_US/i/btn/btn_donate_LG.gif"/></a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)